src/main/java/com/epam/repository/TourOfferDAO.java

```diff
@@ -77,6 +81,35 @@ public TourOffer getTourById(Integer tourId){
         return JdbcTemplate.queryForObject ("SELECT * FROM "+tableName+" WHERE id = ?", parameters, (rs, rowNum) -> buildTour(rs));
     }
 
+    public int amountOfToursSearched(List<Integer> listOfHotelsId, LocalDate startDate, LocalDate endDate){
+        String requestSQL = "SELECT COUNT(*) FROM "+tableName+" WHERE start_date ";
```

Comment:
please, use property variables here, to avoid sqlExceptions in future

Reply:
Done